test/template_test.rb

```diff
@@ -15,6 +15,6 @@ def test_renders_basic_html
   def test_replaces_tag_content
     source = '<div data-prop="title">An Title</div>'
     output = render(source, 'title' => 'Ratchet')
-    assert_equal '<div>Ratchet</div>', output
+    assert_equal '<div data-prop="title">Ratchet</div>', output
```

Comment:
Yes, I'm staying super pragmatic in building this out. So far I don't even mess with attributes other than to look up the property. In this case, it was originally missing in an earlier commit, because I wasn't preserving attributes at all at the time!